example/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.10)
```

Comment:
Thanks for moving this into the example subdirectory. It looks like we still have the example application code in the top level CMakeLists, too. Maybe we can delete that now?

Reply:
Ohhh I deleted it initially but got it back due to a rebase conflict... Good catch.

Reply:
Fixed now